Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyComboBox.cxx

```diff
@@ -469,6 +471,60 @@ void qMRMLSubjectHierarchyComboBox::setAlignPopupVertically(bool align)
   d->AlignPopupVertically = align;
 }
 
+//--------------------------------------------------------------------------
+bool qMRMLSubjectHierarchyComboBox::noneItemEnabled()const
+{
+  if (!this->model())
+    {
+    return false;
+    }
+  return this->model()->noneItemEnabled();
+}
+
+//--------------------------------------------------------------------------
+void qMRMLSubjectHierarchyComboBox::setNoneItemEnabled(bool enable)
+{
+  if (!this->model())
+    {
+    return;
+    }
+  this->model()->setNoneItemEnabled(enable);
+}
+
+//--------------------------------------------------------------------------
+QString qMRMLSubjectHierarchyComboBox::noneItemText()const
```

Comment:
It would be better to mirror qMRMLNodeComboBox API -> rename to `noneDisplay`.

Reply:
First I followed these function names but then I thought that although the `noneEnabled` and `noneDisplay` are unambiguous in a combobox, they are (especially `noneDisplay`) not very expressive of what they does in a subject hierarchy model or tree view. Because I had to introduce the same properties in these two other classes as well. I'll wait for your answer before changing these.

Reply:
I don't particularly like the existing names but it is important to make this new combobox a drop-in replacement of vtkMRMLNodeComboBox. If you want, you can add alternative names (different methods all calling the same underlying method).

Reply:
That's a good point about trivial switch from qMRMLNodeComboBox. I changed these properties and functions back.